app/views/profiles/show.html.erb

```diff
@@ -1,21 +1,17 @@
 <div class="container flex">
-  
+
   <div class="infos">
     <div class="img-container">
-      <% if @profile.profile_pic? %>
         <%= cl_image_tag @profile.profile_pic, class: "image"%>
```

Comment:
You should set class to "avatar-big" (already defined in stylesheets/components/picture) and define width and height